web/web_test.go

```diff
@@ -220,11 +220,8 @@ func TestStatic(t *testing.T) {
 
 	resp, err := http.Get(URL + "/static/root.html")
 
-	if err != nil {
-		t.Fatalf("got error while trying to get static files %v", err)
-	} else if resp.StatusCode != http.StatusOK {
-		t.Fatalf("couldn't get static files %v", resp.StatusCode)
-	}
+	assert.NoErrorf(t, err, "got error while trying to get static files %v", err)
```

Comment:
This line should use `require` as `resp` would be `nil` if an error is returned:

```suggestion
	require.NoErrorf(t, err, "got error while trying to get static files %v", err)
```